readme.md

```diff
@@ -124,9 +116,13 @@ And the complete list of configurable properties:
  * (optional) `gc` override the default GC settings. Available GC settings: `throughput` (default), `hybrid`, `response-time` and `response-time-11`.
  * (optional) `addJava8GCLogging` add java 8 specific gc logging options.
 
-If there is a jar recommending a version of a dependency but that product is not listed in a `productDependency` block an error will be produced: `The following products are recommended as dependencies but do not appear in the product dependencies or product dependencies ignored list: ...`.
+If there is a jar recommending a version of a dependency, that version constraint will be picked up and published in the
+service's manifest, even if that product is not listed in a `productDependency` block. 
+
+It's possible to further restrict the acceptable version range for a dependency by declaring a tighter constraint in a 
+`productDependency` block - this will be merged with any constraints detected from other jars.
```

Comment:
What happens if the ranges don’t overlap?

Reply:
You get an error. I can mention that too.